pkg/build/controller/buildconfig/buildconfig_controller_test.go

```diff
@@ -286,9 +293,25 @@ func buildConfigWithImageChangeTriggers(triggers []tagTriggerID) *buildv1.BuildC
 	return bc
 }
 
+func buildConfigWithImageChangeTriggerStatuses(triggers []tagTriggerID, bc *buildv1.BuildConfig) *buildv1.BuildConfig {
```

Comment:
If this takes a pointer, and returns a pointer, can't you eliminate the returned pointer and just modify the one that you passed into the function?

Reply:
actually at this point I think we can combine `buildConfigWithImageChangeTriggers` and `buildConfigWithImageChangeTriggersStatuses` .... I'll just do that instead